ansible_navigator/actions/config.py

```diff
@@ -223,6 +226,8 @@ def _run_runner(self) -> None:
 
             if self._args.help_config is True:
                 pass_through_arg.append("--help")
+            elif isinstance(self._args.config, str):
```

Comment:
this can be an `if` condition instead of `elif`

Reply:
got it, thx